include/deal.II/hp/dof_handler.h

```diff
@@ -479,6 +479,19 @@ namespace hp
     void
     get_active_fe_indices(std::vector<unsigned int> &active_fe_indices) const;
 
+    /**
+     * Coarsen and refine the finite elements associated with each cell
+     * according to refinement and coarsening flags set and the hierarchy
+     * of the registered hp::FECollection.
+     *
+     * Since the current processor only has control over those cells it owns
+     * (i.e. the ones for which <code>cell-@>subdomain_id() ==
+     * this-@>locally_owned_subdomain()</code>), refinement and coarsening
+     * flags are only respected for those locally owned cells.
+     */
+    void
+    execute_coarsening_and_refinement();
```

Comment:
maybe `execute_p_coarsening_and_refinement` ?

Reply:
Good call. This would prevent confusion with the `Triangulation` equivalent.